render-usage-receipt.php

```
@@ -0,0 +1,96 @@
+
+        // exit immediately to avoid tearDown issues when running outside PHPUnit
+        exit(0);
+    }
+
+    protected function renderPdf(): void
+    {
+        // create a usage receipt with sample data
+        $usageReceipt = \SimpleParkBv\Invoices\Models\UsageReceipt::make();
+
+        // set buyer
+        $buyer = \SimpleParkBv\Invoices\Models\Buyer::make();
```

PHPMD (2.15.0)

Avoid using static access to class '\SimpleParkBv\Invoices\Models\Buyer' in method 'renderPdf'. (undefined)

(StaticAccess)